addons/bestja_page_fixtures_fpbz/templates/pages/privacy_policy.xml

```diff
@@ -32,12 +37,9 @@
                                         </li>
                                     </ul>
                                 </div>
-                                <div class="col-md-9 mb32">
-                                    <h1 class="text-center mt48 mb48 section_title">
-                                        <strong>POLITYKA PRYWATNOŚCI APLIKACJI DO REKRUTOWANIA WOLONTARIUSZY</strong>
-                                    </h1>
+                                <div class="col-md-9 col-md-offset-3 mb32">
                                     <h2 id="wstep" class="mt48 mb16">
-                                        <strong>Wstęp</strong>
+                                        <strong>WSTĘP</strong>
```

Comment:
Change was made actually because the other headers contain great letters. Should I make them all like 'Wstęp' or could we leave this like it is? Thank you.